containers/src/Kokkos_DualView.hpp

```diff
@@ -91,6 +91,25 @@ namespace Kokkos {
  *     behavior.  Please see the documentation of Kokkos::View for
  *     examples.  The default suffices for most users.
  */
+
+namespace Impl {
+
+#ifdef KOKKOS_ENABLE_CUDA
+
+inline const Kokkos::Cuda& get_cuda_space(const Kokkos::Cuda& in) { return in; }
+
+inline Kokkos::Cuda get_cuda_space() {
+  return Kokkos::Cuda(Kokkos::Impl::cuda_get_deep_copy_stream());
```

Comment:
Any reason not to return a default constructed execution space here (which might be slightly cheaper)?

Reply:
So, if we did a default constructed space, we're enqueueing in the default stream, meaning that things in the default stream have to wait on these operations. This way, these copies don't interfere.

Reply:
OK.